examples/experimental/dagster-airlift/README.md

```diff
@@ -101,63 +114,96 @@ defs = Definitions.merge(
 
-```python
-from dagster_airlift.def_factory import DefsFactory
+### Mapping assets to tasks
 
-@dataclass
-class DbtProjectDefs(DefsFactory):
-    dbt_project_path: Path
-    name: str
-    group: Optional[str] = None
+The `from_task` and `from_dag` utilities link created assets to their underlying tasks. Assets which are properly linked will be materialized by the Airlift sensor once the corresponding task completes.
 
-    def build_defs(self) -> Definitions:
-        dbt_manifest_path = self.dbt_project_path / "target" / "manifest.json"
+### Viewing observed assets
 
-        @dbt_assets(manifest=json.loads(dbt_manifest_path.read_text()), name=self.name)
-        def _dbt_asset(context: AssetExecutionContext, dbt: DbtCliResource):
-            yield from dbt.cli(["build"], context=context).stream()
+Once your assets are set up, you should be able to reload your Dagster definitions and see a full representation of the dbt project and other data assets in your code.
 
-        if self.group:
-            _dbt_asset = _dbt_asset.with_attributes(
-                group_names_by_key={key: self.group for key in _dbt_asset.keys}
-            )
+Kicking off a run of the DAG, you should see the newly created assets materialize in Dagster.
 
-        return Definitions(
-            assets=[_dbt_asset],
-            resources={
-                "dbt": DbtCliResource(
-                    project_dir=self.dbt_project_path, profiles_dir=self.dbt_project_path
-                )
-            },
-        )
-```
+_Note: There will be some delay between task completion and assets materializing in Dagster, managed by the sensor. This sensor runs every 30 seconds by default (you can reduce down to one second via the `minimum_interval_seconds` argument to `sensor`), so there will be some delay._
+
+## Migrating Assets
+
+Once you have created corresponding definitions in Dagster to your Airflow tasks, you can begin to selectively migrate execution of some or all of these assets to Dagster.
+
+To begin migration on a DAG, first you will need a file to track migration progress. In your Airflow DAG directory, create a `migration_state` folder, and in it create a yaml file with the same name as your DAG. The included example at [`examples/tutorial-example/airflow_dags/migration_state`](.examples/tutorial-example/airflow_dags/migration_state) can be used as reference.
 
-We imagine most people will have to customize this, so we encourage you to copy and paste it for now.
+Given our example DAG `rebuild_customers_list` with three tasks, `load_raw_customers`, `run_dbt_model`, and `export_customers`, `migration_state/rebuild_customers_list.yaml` should look like the following:
 
-Then you can add instances of the Factory to your `build_defs_from_airflow_instance` call:
+```yaml
+tasks:
+  load_raw_customers:
+    migrated: False
+  run_dbt_model:
+    migrated: False
+  export_customers:
+    migrated: False
+```
```

Comment:
@dpeng817 

Just a heads up I want to change this to 

```yaml
tasks:
  - id: load_raw_customers
    migrated: False
```

We don't want to be subject to whatever rules there are for keys in yaml. Not sure what task id rules are across all flavors of Airflow.

Reply:
@dpeng817